### PR TITLE
Added hubert schmid gmbh

### PR DESCRIPTION
**Pull request by MichaelAlt**

### CODE REVIEW

**Review by 5ila5 — CHANGES_REQUESTED** [review]

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/api_hubert_schmid_de.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+import requests
+from waste_collection_schedule import Collection  # type: ignore[attr-defined]
+
+TITLE = "HubertSchmid Recycling und Umweltschutz GmbH"
+DESCRIPTION = "Abfuhrtermine Blaue Tonne"
+URL = "https://www.hschmid24.de/BlaueTonne/"
+API_URL = "https://www.hschmid24.de/BlaueTonne/php/ajax.php"
+
```
[the remark this answers is not in the document]
Comment (MichaelAlt) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/api_hubert_schmid_de.py:10]: done

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/api_hubert_schmid_de.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+class Source:
+    def __init__(
+        self,
+        city: str | None = None,
```
[the remark this answers is not in the document]
Comment (MichaelAlt) [reply custom_components/waste_collection_schedule/waste_collection_schedule/source/api_hubert_schmid_de.py]: yes actually city is the only required attribute.

File: doc/source/api_hubert_schmid_de.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        
+```
+
+### Configuration Variables
+
+**city**  
+*(String) (required)*
+
+**strasse**  
+*(String) (required)*
```
[the remark this answers is not in the document]
Comment (MichaelAlt) [reply doc/source/api_hubert_schmid_de.md]: yes correct, fixed it.

File: doc/source/api_hubert_schmid_de.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# Apps by Abfall+
```
[the remark this answers is not in the document]
Comment (MichaelAlt) [reply doc/source/api_hubert_schmid_de.md]: fixed

**Review by 5ila5 — APPROVED** [review]

**Pull request merged**